incubator/virtualcluster/pkg/syncer/resources/crd/checker.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```

Comment:
Please change the copyright year. 

As you may notice, we have switched to the new differ framework to replace the old patroller logic.
sigs.k8s.io/multi-tenancy/incubator/virtualcluster/pkg/syncer/patrol/differ/differ.go

Can you please adopt the same framework?

Reply:
Sure